config/software/automake.rb

```diff
@@ -38,7 +35,7 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
-  if version.satisfies?(">= 1.15")
+  if version.satisfies?(">= 1.15") && version.satisfies?("< 1.16")
```

Comment:
```suggestion
  if version == "1.15"
```

This will catch that in a simpler way